### PR TITLE
LPS-56065-PortalSettingsPortletToOSGI2 struts to MVC

### PR DESCRIPTION
**Pull request by philipjones03**
@peterfellwock fyi

### CODE REVIEW

**Comment by peterfellwock** [comment]
ci:retest

**Comment by rotty3000** [comment]
I'm not sure what's going on.. but I don't think this failure is on the
part of Phil!

**Comment by rotty3000** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: portal-web/docroot/WEB-INF/liferay-portlet.xml [hunk 1]
```diff
@@ -554,12 +554,10 @@
 	<portlet>
```
Comment (rotty3000) [inline portal-web/docroot/WEB-INF/liferay-portlet.xml]: this whole portlet section should be removed.. you're confusing the portal.

File: portal-web/docroot/WEB-INF/portlet-custom.xml [hunk 2]
```diff
@@ -714,14 +714,14 @@
 	<portlet>
```
Comment (rotty3000) [inline portal-web/docroot/WEB-INF/portlet-custom.xml]: this whole portlet section should also be removed.

File: modules/apps/portal-settings/portal-settings-web/src/META-INF/resources/init.jsp [hunk 3]
```diff
@@ -14,7 +14,7 @@
  */
 --%>
 
-<%@ include file="/html/portlet/init.jsp" %>
+<%@ include file="/portlet/init.jsp" %>
```
Comment (rotty3000) [inline modules/apps/portal-settings/portal-settings-web/src/META-INF/resources/init.jsp]: this file doesn't exist, nor should it. Can remove this line and embed the parts you need from that file in core.

**Pull request closed without merging**